app/controllers/users_controller.rb

```diff
@@ -77,7 +77,7 @@ def changed_username
     @new_login = params[:new_login]
 
     unless logged_in_as_admin? || @user.valid_password?(params[:password])
-      flash[:error] = t(".user.incorrect_password")
+      flash[:error] = t(".user.incorrect_password_html", contact_support_link: helpers.link_to(t("users.confirm_change_email.contact_support"), new_feedback_report_path))
```

Comment:
Please use a locale key for the link caption that is relative to this string, e.g. `.user.contact_support` so that it is separate from the other error message